libraries/standard/mqtt/include/mqtt_state.h

```diff
@@ -26,6 +26,13 @@
 
 #define MQTT_STATE_CURSOR_INITIALIZER    ( size_t ) 0
 
+/**
+ * @brief Invalid packet identifier.
+ *
+ * Zero is an invalid packet identifier as per MQTT v3.1.1 spec.
+ */
+#define MQTT_PACKET_ID_INVALID           ( ( uint16_t ) 0U )
```

Comment:
I know this was in `mqtt_state.c` before, but it seems like it would make more sense to be in `mqtt.h` instead of `mqtt_state.h`